openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/AddIngestion.component.tsx

```diff
@@ -214,19 +214,23 @@ const AddIngestion = ({
   const getFilterPatternData = (data: FilterPattern) => {
     const { includes, excludes } = data;
 
-    return isUndefined(includes) && isUndefined(excludes)
-      ? undefined
-      : {
-          includes: includes && includes.length > 0 ? includes : undefined,
-          excludes: excludes && excludes.length > 0 ? excludes : undefined,
-        };
+    const filterPattern =
+      (!isUndefined(includes) && includes.length) ||
+      (!isUndefined(excludes) && excludes.length)
+        ? {
+            includes: includes && includes.length > 0 ? includes : undefined,
+            excludes: excludes && excludes.length > 0 ? excludes : undefined,
+          }
+        : undefined;
+
+    return isEmpty(filterPattern) ? undefined : filterPattern;
```

Comment:
@ShaileshParmar11, Do we need to check for `isEmpty` because `filterPattern` will be undefined not an empty object.